src/main/groovy/com/jfrog/bintray/gradle/BintrayUploadTask.groovy

```diff
@@ -473,6 +470,21 @@ class BintrayUploadTask extends DefaultTask {
         }
     }
 
+    def assertUploadConfiguration() {
+        def errors = []
+        if(user == null) {
+            errors.add("user is not configured")
+        }
+        if(apiKey == null) {
+            errors.add("key is not configured")
+        }
+        if(!errors.isEmpty()) {
+            errors.join(" and ")
```

Comment:
It is used, see `should throw an exception when user and api key is not configured` test.